vuepress/docs/next/docs/consume/entandoapp-cr.md

```diff
@@ -0,0 +1,39 @@
+---
+sidebarDepth: 2
+---
+
+# EntandoApp Custom Resource Definition 
+The EntandoApp CRD is the deployment of a Docker image that hosts the Entando and Java-based web application. Server-side components include the Entando App Engine,  Entando Component Manager, Entando App Builder, and the user-facing application.
+ 
+Entando offers standard WildFly or EAP images for the definition, but typically customers provide their own.
+
+## Example EntandoApp
+ 
+```yaml
+apiVersion: entando.org/v1
+kind: EntandoApp
+metadata:
+  namespace: entando
+  name: my-app    
+spec:
+  environmentVariables: []
+  dbms: embedded
+  ingressHostName: my-app.192.168.64.5.nip.io
+  standardServerImage: eap
+  replicas: 1
+ ```
+## Specifications 
+| Spec Name | Description |
+| :- | :- |
+| `spec.customServerImage`|  Used to deploy the Docker image containing your custom Entando App. Follow these instructions on how to [build your own image](../../tutorials/devops/build-core-image.md). This property and the `spec.standardServerImage` are  mutually exclusive.|
+|`spec.dbms` | Allowed values are: MySQL, PostgreSQL (default), Oracle, or embedded. Oracle is only supported as an external database.|
+|`spec.ecrGitSshSecretName`| The configuration used by the Entando Component Repository to download bundles from authenticated Git repositories. It's a secret containing a private key file named `rsa_id` that matches a public key configured in the authenticated Git repository.|
+|`spec.environmentVariables`| A map of environment variables to pass to the EntandoApp Docker image. This can be used to provide connection details for custom datasources or message queues as discussed in the [custom datasources tutorial](../../tutorials/devops/change-default-datasource.md). These variables can sometimes be used as a mechanism to override any of the default environment variables that need customization.|
```

Comment:
connection details *of* custom datasources...

omit "sometimes" - "can" renders "sometimes" unnecessary